client/my-sites/checkout/composite-checkout/components/wp-checkout.tsx

```diff
@@ -680,6 +680,13 @@ const SubmitButtonFooter = () => {
 				{ translate( '7 day money back guarantee on monthly subscriptions' ) }
 			</>
 		);
+
+	const disclaimerContent = responseCart?.products?.some( ( product ) => {
+		return !! product?.introductory_offer_terms?.enabled;
+	} )
+		? translate( '* All discounts are for the first term only, renewals are at full price.' )
```

Comment:
It might be worth adding a translator comment to this line as well maybe? I don't know how other languages deal with footnotes/asterisks but I figure a translator might think this asterisk is just a bullet point or something. 🤷

Reply:
Of course! Should have thought of that myself haha. Thank you again